cfme/tests/services/test_provision_stack.py

```diff
@@ -127,7 +131,16 @@ def catalog():
     catalog = Catalog(name=cat_name, description="my catalog")
     catalog.create()
     yield catalog
-    catalog.delete()
+
+
+@pytest.fixture(scope="function")
+def set_grid_view():
```

Comment:
We should perhaps have a function inthe configure module

```python
def set_view(button_group_name, view):
    bg = ButtonGroup(button_group_name)
    sel.force_navigate("my_settings_default_views")
    default_view = bg.active
    if(default_view != view):
        bg.choose(view)
        sel.click(form_buttons.save)

set_view('Stacks', 'Grid View')
```
@akarol can you comment on this please?